src/components/Navbar/Navbar.tsx

```diff
@@ -29,16 +34,23 @@ export const Navbar = ({ disableNavbar = false }) => {
     if (pathname === '/gas/') {
       return;
     }
-
-    isLearnPage ? router.push(JUMPER_LEARN_PATH) : router.push('/');
+    if (isLearnPage) {
```

Comment:
are we sure about this? that means than people cannot really go back to main page? :(

Reply:
There is a big button for this. Like on Jumper Learn.